### DISCUSSION TITLE
Is it possible to use a pre-generated image with this tool?

### DISCUSSION THREAD

**Discussion by dpatey** [BalintKomjati/smooth-infinite-zoom#4] [General]
I thought it might be as simple as assigning a local file path to the current_image variable, but it seems to not be the case and I can't quite see where I'm going wrong. 

Either way, really nice job on this notebook, @BalintKomjati !

**Comment by dpatey** [comment]
Crap just saw your comment in the other thread - will try messing with the init_images!

**Reply by BalintKomjati** [reply]
many people request this so will pull sthing together 
in fact there is a branch already for this (you will find by the name right away) where this already works (but not 1st draft only, not very user friendly) @dpatey
